server/views/layout/default.njk

```diff
@@ -21,7 +21,7 @@
     <link rel="manifest" href="{{ '/assets/icons/' | getHashedFile('manifest.json') }}'"/>
     <link rel="mask-icon" href="{{ '/assets/icons/' | getHashedFile('safari-pinned-tab.svg') }}" color="#000000"/>
     <script src="{{ '/assets/libs/' | getHashedFile('picturefill.min.js') }}" async></script>
-    <script async defer src="//platform.twitter.com/widgets.js"></script>
+
```

Comment:
Did this not also handle the WordPress in-article tweets? If so, are we happy for those to stop rendering with Twitter styles.

Reply:
https://wellcomecollection.org/articles/no-photography/